sites/public/pages/listings/index.tsx

```diff
@@ -0,0 +1,112 @@
+import Head from "next/head"
```

Comment:
QQ: is it possible to have a `pages/listings.tsx` as well as `pages/listings/filtered.tsx`? If we move the `/listings` to this index, we'd no longer be tracking with any upstream changes to the `/listings` page.

Reply:
Done